htdocs/addtolist.php

```diff
@@ -31,7 +31,7 @@
 }
 $newlist_public = isset($_REQUEST['newlist_public']);
 $newlist_watch = isset($_REQUEST['newlist_watch']);
-$default_list = isset($_REQUEST['listid']) ? (int) $_REQUEST['listid'] : cachelist::getMyLastAddedToListId();
+$default_list = isset($_REQUEST['listid']) ? (int)$_REQUEST['listid'] : cachelist::getMyLastAddedToListId();
```

Comment:
Auch solche Konstrukte wurden kürzlich in die andere Richtung geändert - Space eingefügt nach Typecast. Denke das ist ein Fall für eine Diskussion / Festlegung in unseren Style Guides.